lxcfs.c

```diff
@@ -82,22 +186,27 @@ static void do_reload(void)
 		goto good;
 
 	dlopen_handle = dlopen("/usr/lib/lxcfs/liblxcfs.so", RTLD_LAZY);
```

Comment:
I'm happy to simplify this wherever possible. Maybe we can simplify keeping track of the namespace we are currently in aka avoiding the use of a struct for this.